lib/rouge/lexers/html.rb

```diff
@@ -27,7 +27,7 @@ def self.analyze_text(text)
         rule /<!--/, Comment, :comment
         rule /<\?.*?\?>/m, Comment::Preproc # php? really?
 
-        rule /<\s*script\s*/m do
+        rule /<\s*script\s*(?!\s*\D*type=\"text\/(x-tpl|template)\")/m do
```

Comment:
Can we use a new state here and check the actual value of the attribute rather than this complicated negative lookahead?